output/request_printer.go

```diff
@@ -36,6 +36,18 @@ func PrintRequest(request request.Request) error {
 		}
 	}
 
+	if len(request.Cookies) > 0 {
+		sentCookieKeyColor := color.New(color.Bold, color.FgBlue).PrintfFunc()
```

Comment:
Potentially move each type of formatting out to it's own helper method?
```golang
func PrintRequest(request request.Request) error {
	// other implementation
	if len(request.Cookies) > 0 {
      printCookies(request.Cookies); 
    }
   // other implementation
}

func printCookies(c []Cookie) error {
// some implementation
}
```
Should make it easier to test / understand intent of?

Reply:
I agree that it would make it easier to read. I think the intent is pretty clear from the name of the method.

I don't think it would make it easier to test, unless I was willing to make all of the public (I don't like to test private methods).

I'll break it apart since I don't have reasons not too besides being lazy. :-)

Reply:
I meant intent of each block of code within that method - I agree, the `PrintRequest` method is pretty explicit (though maybe `PrintRequestDetails` would be even more precise? I don't think it really matters.)

This is really more of a nit-pick since the additional printing logic you added is not that hard to reason about.

I was just thinking more down the line where there's could be potentially a lot of different printing logic depending on what the user would like outputted.